Source/WTF/wtf/posix/ThreadingPOSIX.cpp

```diff
@@ -53,12 +53,6 @@
 #endif
 #endif
 
-#if !COMPILER(MSVC)
-#include <limits.h>
-#include <sched.h>
-#include <sys/time.h>
-#endif
```

Comment:
Hmm, this is a `!` but I take it we don't need to keep them? (I guess `<sched.h>` is already included under `OS(LINUX)`...)

Reply:
This is ThreadingPOSIX.cpp, so it is not used in Windows.

Reply:
Of course; I'm saying the `#if` has a `!`.